spec/controllers/bills_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+describe BillsController do
+
+  describe "GET #index" do
+    let!(:bill) { FactoryGirl.create(:bill) }
```

Comment:
could you put it in a `before` statement outside the current `describe`?
also, does it need to be `let!` as opposed to just `let`?